storage/blob.go

```diff
@@ -6,13 +6,42 @@ package storage
 import (
```

Comment:
I'm wondering whether this `storage/blob.go` should now move to `storagenode/...` somewhere, since this isn't a general purpose blob handling thing.

Reply:
I would be fine with moving it, but I don't see any parts that don't look like general purpose blob handling. A general purpose blob store can have multiple storage schemes.

Reply:
Yeah, I guess so. To me the difference is that there would be multiple general purpose blob store each having their own storage schemes. At least that would keep the logic and interfaces simpler.

Here the FromatV0/FormatV1 are describing things that seem tied to a particular blob store.

Internally, sure it might contain different versions, but it likely wouldn't expose them directly via the "general purpose" interface.

But, yeah, anyways, let's leave it as is for this PR. Otherwise it'll get confusing in what has been reviewed and what not.

Reply:
That's a good point. I'll move the FormatVersion-specific things. I already need to make a not-insignificant number of minor changes of that sort, so confusion about what has been reviewed is already going to be a problem :(